modules/acquia_cms_common/src/Facade/ConfigHandlerFacade.php

```diff
@@ -120,4 +120,25 @@ public function processThirdPartySettings(array $entity, array $settings): void
     }
   }
 
+  /**
+   * Update config on the basis of key.
+   *
+   * @param string $config
+   *   Config settings data.
+   * @param string $key
+   *   Configuration key which needs to be updated.
+   * @param array $value
+   *   Configuration data needs to be updated.
+   *
+   * @return void
+   *   Saving the configuration.
+   */
+  public function updateConfigByKey(string $config, string $key, array $value): void {
+    $config = $this->configFactory->getEditable($config);
+    if ($config->get($key)) {
+      $config->set($key, $value);
+    }
+    $config->save();
```

Comment:
Why this `$config->save();` is out side of if statement?

Reply:
Can't we use existing function `_acquia_cms_common_update_page_configurations()` for this purpose?

Reply:
Idea is to use configHandlerFacade class to update the config related data.